insights/parsers/named_conf.py

```diff
@@ -16,31 +16,44 @@
 class NamedConf(NamedCheckconf):
     """
     Class for parsing the file ``/etc/named.conf```, We use class ``NamedCheckConf`` to parse most
-    of the named.conf configurations and class ``NamedConf`` to parse the `include` directives.
+    of the named.conf configurations and class ``NamedConf`` to parse the `include` directives and
+    the `allow-recursion` value from the `options` section.
 
     .. note::
         Please refer to the super-class :py:class:`insights.parsers.named_checkconf:NamedCheckConf`
         for more usage information.
 
     Attributes:
-        includes (list): List of files in 'include' section.
+        includes (list): List of files in 'include' and 'allow-recursion' value from the 'options' section.
```

Comment:
I think it's a new attribute that is at the same level as the `self.includes`, you'd add it as a new attribute.

Reply:
Correct, updated related to it.